modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/configurators/RootProjectConfigurator.java

```diff
@@ -600,59 +596,6 @@ public void execute(Task task) {
 		return dockerfile;
 	}
 
-	private CreateTokenTask _addTaskCreateToken(
-		Project project, final WorkspaceExtension workspaceExtension) {
-
-		CreateTokenTask createTokenTask = GradleUtil.addTask(
-			project, CREATE_TOKEN_TASK_NAME, CreateTokenTask.class);
-
-		createTokenTask.setDescription("Creates a liferay.com download token.");
-
-		createTokenTask.setEmailAddress(
```

Comment:
You can remove all of this configuration though, we can just add the task but leave off all the field.